doc/3/controllers/index.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This file, and other documentation files, should be created in the `doc/4` directory instead

Reply:
In deed
Just, since it doesnt exist in the documentation framework (https://github.com/kuzzleio/documentation/tree/master/src/sdk/php)
commands as `doc-prepare` should fail, so the Dead Link check also

Reply:
So you have to add it